index.html

```diff
@@ -182,17 +182,18 @@ <h4 class="ab-titles skills"> Skills</h4>
       <div class="infography-contact">
         <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
       </div>
-      <div class="form flex-column">
-        <form class="flex-column " action="https://formspree.io/f/mayzgdra" method="POST" class="form-data">
+      <div class="flex-column">
+        <form id="form" class="flex-column form-data" action="'https://formspree.io/f/mayzgdra'" method="POST">
           <input type="text" name="name" id="name" placeholder="Full name" required maxlength="30">
-          <div class="first-last-name">
+          <!-- <div class="first-last-name">
             <input type="text" name="first-name" id="first-name" placeholder="First Name" required maxlength="30">
             <input type="text" name="last-name" id="last-name" placeholder="Last Name" required maxlength="30">
-          </div>
+          </div> -->
```

Comment:
- [ ] You have done great job :clap: However, Please do not commit old pieces of code as inline comments. Leaving old pieces of code as comments can clutter the codebase and make it harder to read and understand. Over time, as more code comments accumulate, it becomes challenging to differentiate between actual code and commented-out sections. This clutter can hinder the development process and introduce confusion for developers trying to navigate and maintain the code. :slightly_smiling_face: